exercises/practice/affine-cipher/.docs/instructions.append.md

```diff
@@ -0,0 +1,37 @@
+# Hints
+
+You need to implement the `decode` and `encode` functions, which decode and encode a `String` using an Affine cipher.
+You can use the provided signature if you are unsure about the types, but don't let it restrict your creativity.
+
+This exercise works with textual data. For historical reasons, Haskell's
+`String` type is synonymous with `[Char]`, a list of characters. For more
+efficient handling of textual data, the `Text` type can be used.
+
+As an optional extension to this exercise, you can
+
+- read about [string types](https://haskell-lang.org/tutorial/string-types) in
```

Comment:
haskell-lang.org seems dead. Perhaps the Archive has a copy?

Reply:
I think we can remove this link as there is a link to `Data.Text` later on the page, which is IMHO sufficient.